docs/using-jda/interactions.md

```diff
@@ -170,6 +170,32 @@ When you use `deferReply` the first message sent to this webhook will act identi
         }
         ```
 
+#### Managing Slash Command Permissions
+
+Restricting slash commands to specific roles or members is up to the admins of the guild.  However, bots can specify a set of permissions that sets who can use those commands by default.
+
+Note that members with `Permission.ADMINISTRATOR` can always use commands, and they can override which roles or members can run the command.  JDA only changes the _default_ members and roles that can access it.
```

Comment:
I'm a bit confused about those double spaces here.

Wouldn't it be better in terms of readability to use a `<br>` tag (So that it's clear in the raw view that it's a line break) and have the following text on the next line?